xbmc/windowing/ios/WinSystemIOS.mm

```diff
@@ -498,3 +503,18 @@ - (void) runDisplayLink
 {
   return m_winEvents->MessagePump();
 }
+
+void* CWinSystemIOS::GetProcAddressGL(const char* name)
+{
+  if (!m_glLibrary)
+  {
+    const char* glLibPath = "/System/Library/Frameworks/OpenGLES.framework/OpenGLES";
+
+    m_glLibrary = dlopen(glLibPath, RTLD_LAZY);
```

Comment:
I don't have a lot of experience with OSX/iOS, does this work with it and there is no extension on path?

Reply:
It does, they have some sort of dynamic loading on certain paths is my understanding

Reply:
It's strange to me also, I just did a quick google and did some test code on my wifes macbook. No idea if this actually works on iOS/TVOS though (needs to be tested).